src/main/java/com/cegeka/academy/service/invitation/InvitationServiceImpl.java

```diff
@@ -59,6 +70,36 @@ public void deleteInvitationById(Long id) {
         invitationRepository.findById(id).ifPresent(invitation -> invitationRepository.delete(invitation));
     }
 
+    @Override
+    public void sendGroupInvitationsToPrivateEvents(Long idGroup, Invitation invitation) throws NotFoundException {
+
+        if (invitation.getEvent() != null) {
+
+            if (invitation.getEvent().getPublic().equals(false)) {
```

Comment:
Observatiile de mai jos sunt pentru "estetica" si nu e obligatoriu sa le implementezi.
De regula pentru getter-ii care returneaza booleani este inlocuti prefixul `get` cu `is`.
De exemplu:
`getPublic()` devine `isPublic()`

Daca vrei sa verifici intr-un `if` un boolean atunci pur si simplu evaluezi booleanul respectiv.